homeassistant/components/climate/eq3btsmart.py

```diff
@@ -60,8 +60,11 @@ def target_temperature(self):
         """Return the temperature we try to reach."""
         return self._thermostat.target_temperature
 
-    def set_temperature(self, temperature):
+    def set_temperature(self, **kwargs):
         """Set new target temperature."""
+        temperature = kwargs.get(ATTR_TEMPERATURE)
+        if temperature is None:
```

Comment:
It actually can:
If a user uses a script or developer tools in HA to set target_temperature_high and target_temperature_low to a platform that does not have those items, it will pass in those two, bt not the temperature attribute, and it will be None, even though we have validation, it is allowed to pass only temperature or only target_temp_high and target_temp_low.